csm/core/services/usl.py

```diff
@@ -481,12 +467,11 @@ async def post_register_device(
             await self._cleanup(
                 s3_account_service,
                 s3_iam_users_service,
-                s3_access_keys_service,
                 s3_bucket_service,
                 lyve_pilot_s3_credentials,
                 lyve_pilot_s3_account.get('account_name'),
                 lyve_pilot_iam_user.get('user_name'),
-                lyve_pilot_iam_user_access_key.get('access_key_id'),
+                lyve_pilot_iam_user.get('access_key_id'),
```

Comment:
We have to remove this paramater from this call as well (if it is removed from the `_cleanup` function impl).

Reply:
Please check line 529 of this commit, too. I can't leave a comment there for some reason.

Reply:
(These comments should be one line above 🙃)

Reply:
Removed.